.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: "poetry"
       - name: lint codebase
-        run: poetry run flake8 --ignore=E501,E402 .
+        run: poetry run flake8 --ignore=E501,E402,E275 .
```

Comment:
This commit message should say what E275 is and why we ignore it

Reply:
commit message updated!